plugins.json

```diff
@@ -931,7 +931,7 @@
         "description": "This plugin gives the ability to assign actual Task (direct in comment-create section) to one of the users in Project.",
         "homepage": "https://github.com/ipunkt/KanboardCommentActions",
         "readme": "https://github.com/ipunkt/KanboardCommentActions/blob/master/README.md",
-        "download": "https://github.com/ipunkt/KanboardCommentActions/releases/download/1.0.0/KanboardCommentActions-1.0.0.zip",
+        "download": "https://github.com/ipunkt/KanboardCommentActions/releases/download/v1.0.0/KanboardCommentActions-1.0.0.zip",
```

Comment:
This link is now returning a 404:

```
wget https://github.com/ipunkt/KanboardCommentActions/releases/download/v1.0.0/KanboardCommentActions-1.0.0.zip
--2019-10-28 18:59:36--  https://github.com/ipunkt/KanboardCommentActions/releases/download/v1.0.0/KanboardCommentActions-1.0.0.zip
Resolving github.com (github.com)... 192.30.253.113
Connecting to github.com (github.com)|192.30.253.113|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2019-10-28 18:59:37 ERROR 404: Not Found.```

Reply:
@fguillot ich have tested the link and functional [download](https://github.com/ipunkt/KanboardCommentActions/releases/download/v1.0.0/KanboardCommentActions-1.0.0.zip )
could you please accept the pull request.